### ISSUE TITLE
plot connectivity scales

### ISSUE THREAD

**Issue by etiennedemontalivet** [lmcinnes/umap#1071]
I would like to add extra information on top of the connectivity plot, but it seems that the connectivity plot values do not correspond to the UMAP transform ? 

Here is a reproducible example:

``` python
import umap
import umap.plot
from sklearn.datasets import make_classification

X, _ = make_classification()
random_texts = [str(i) for i in range(X.shape[0])]

mapper = umap.UMAP().fit(X)
projected_X = mapper.transform(X)
ax = umap.plot.connectivity(mapper, edge_bundling='hammer')
for x, y, s in zip(projected_X[:,0], projected_X[:,1], random_texts):
    ax.text(x, y, s)
    break # added this for debug
```
![image](https://github.com/lmcinnes/umap/assets/7569375/8b2e4834-5e7f-49d1-854e-3796ad75c8ef)

Expected behavior: The connectivity plot should return an axis which is comparable with the UMAP transform

Or is there any easy way to get the transformation that is applied ?
Thanks!
